salt/states/cmd.py

```diff
@@ -1041,7 +1041,7 @@ def script(name,
             'Replace them with runas. '
             'These arguments will be removed in Salt Oxygen.'
         )
-        if kwargs['user'] is not None and runas is None:
+        if 'user' in kwargs and kwargs['user'] is not None and runas is None:
```

Comment:
It was just a thought. I can't think of one offhand, since I'd expect strings to be the only case we operate on. I'm fine either way. Just let me know.